src/button/internal.tsx

```diff
@@ -6,6 +6,7 @@ import { fireCancelableEvent, isPlainLeftClick } from '../internal/events';
 import useForwardFocus from '../internal/hooks/forward-focus';
 import styles from './styles.css.js';
 import testUtilStyles from './test-classes/styles.css.js';
+import analyticsLabels from './analytics-metadata/styles.css.js';
```

Comment:
Minor: A better name might be `analyticsStyles` or `analyticsSelectors`

Reply:
Updated